### DISCUSSION TITLE
Beatmap page leaderboard max combo always refers to classic mod maximum

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#21665] [Q&A]
![image](https://user-images.githubusercontent.com/53081750/208117722-ff054717-177f-4324-a0a5-d1050f1bf772.png)
This doesn't happen in the results screen
![image](https://user-images.githubusercontent.com/53081750/208117675-77bad7fd-381a-47c1-a7ed-2cc99e4bd127.png)
2022.1208.0

**Comment by smoogipoo** [comment]
Max combo here is likely coming from the API request beatmap object. Should instead be calculated form the hit statistics.
